FusionMCPBridge/tests/test_live_tool_libraries.py

```
@@ -0,0 +1,326 @@
+                "sort_order": "asc"
+            }
+        )
+        
+        assert not response_is_empty(response), "Advanced search should not be empty"
+
+
+class TestSearchSuggestions:
+    """Tests for /tool-libraries/search/suggestions endpoint."""
+    
+    @pytest.mark.skip(reason="Timeout - bridge slow to respond")
+    def test_suggestions_returns_response(self, bridge_available):
```

Ruff (0.14.10)

Unused method argument: `bridge_available`

(ARG002)